src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -521,6 +526,7 @@ def load_arguments(self, _):
     with self.argument_context('vm image list') as c:
         c.argument('image_location', get_location_type(self.cli_ctx))
         c.argument('edge_zone', edge_zone_type)
+        c.argument('architecture', architecture_type)
```

Comment:
The argument types are abstracted into `CLIArgumentType` only when there are multiple places that need to be reused. Otherwise, it is recommended to directly declare the attributes of parameter
```suggestion
        c.argument('architecture', help='The name of architecture. ', arg_type=get_enum_type(["x64", "Arm64"])
```

Reply:
Done